packages/mobile/src/screens/trending-screen/TrendingScreen.tsx

```diff
@@ -53,6 +54,8 @@ const trendingScreens = [
 ]
 
 export const TrendingScreen = () => {
+  usePopToTopOnDrawerOpen()
```

Comment:
nice, wondering if it makes sense to make this an option for the <Screen /> component?

Reply:
Which component? The `Screen` component?